### ISSUE TITLE
Loggable extensions and user action logs

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#38454]
### Steps to reproduce the issue
In the **user acytion logs** component options remove some or all **events to log**
![image](https://user-images.githubusercontent.com/1296369/184470839-9b322719-00d7-4a39-8014-626cc470ae89.png)

In your user profile go to the Action Log tab and configure the **Events to Email**

![image](https://user-images.githubusercontent.com/1296369/184470883-d53e4b35-3cea-473f-b7cd-abefb1d32bc6.png)

### Expected result
The list of events to email contains a list of **only** the events that are being logged

### Actual result
There is no connection between the list of events to mail and the list of events to log.

### Additional comments
In other words it says it will send you an email on an action in the X component but it will not because you are only logging actions in Y component

**Comment by alikon** [comment]
please test #38588
